spec/counter_spec.rb

```diff
@@ -1,53 +0,0 @@
-require 'spec_helper'
-
-describe Prorate::Counter do
```

Comment:
The `throttle!` method now returns an integer indicating how many more calls you can make before you'll hit the block.